renovate.json

```diff
@@ -17,6 +17,7 @@
     "enabled": false
   },
   "enabledManagers": [
+    "gomod",
```

Comment:
Merging may be the only way to test this, so I'm fine with merging and reverting if needed.